libs/form/src/next/upload/vl-upload.component.ts

```diff
@@ -183,6 +184,7 @@ export class VlUploadComponent extends FormControl {
 
         if (changedProperties.has('maxFiles')) {
             if (this.dropzoneInstance) this.dropzoneInstance.options.maxFiles = this.maxFiles;
+            this.multiple = this.maxFiles > 1;
```

Comment:
Kan je dit in een andere PR doen? Liefst enkel de fix in deze PR.

Reply:
Never mind is nodig voor de parseFormData().
Hoe zit het als de afnemer geen maxFiles definieert maar de gebruiker wel toelaat om meerdere files te uploaden?

Reply:
Ik heb dit overgenomen van vroegere werking; als je bij Dropzone `maxFiles` instelt op meer dan 1, dan wordt het vanzelf een multi-upload. Maar je maakt hier wel een punt dat het natuurlijk wel kan voorvallen dat er meer dan 1 mag geüpload worden maar dat er geen maximum aantal bestanden ingesteld moet worden voor de afnemer.

In dat geval kunnen we ofwel:
- van het `multiple`-attribuut natuurlijk wel een instelbaar attribuut maken (bij `maxFiles > 1` komt dit er dan vanzelf op als het er nog niet op stond)
- nog een ander attribuut voorzien die `uploadMultiple` van Dropzone op true zal zetten

Reply:
Wat gebeurt er als je geen maxFiles instelt? Kan de gebruiker er dan meerdere uploaden?

Reply:
Nee, dan is het standaard maximum 1 bestand.